osu.Game/Overlays/Mods/ModSelectScreen.cs

```diff
@@ -192,7 +192,8 @@ protected override void LoadComplete()
 
             State.BindValueChanged(_ => samplePlaybackDisabled.Value = State.Value == Visibility.Hidden, true);
 
-            ((IBindable<IReadOnlyList<Mod>>)modSettingsArea.SelectedMods).BindTo(SelectedMods);
+            if (customisationButton != null)
+                ((IBindable<IReadOnlyList<Mod>>)modSettingsArea.SelectedMods).BindTo(SelectedMods);
```

Comment:
best to add an inline comment here mentioning that this is a quick optimisation, because on its own it's a bit weird (my immediate thought would be that the `modSettingsArea` shouldn't be present in this case).

Reply:
Depends on what you assume to be "present" I guess. Alpha=0 or Height=0 are the same "presence value" to me (none), which is the default state without a button being present.

Added comment.

Reply:
By present I meant like, the drawable component itself. But I'm fine with this workaround for now as that would require further consideration.